Ubuntu 24.04: Can't build `wayfire-plugins-extra` due to nlohmann-json

I have installed nlohmann-json using homebrew but I get this error when installing Wayfire:

```
+ read -p 'Do you want to install wayfire-plugins-extra? [y/n]? ' yn
Do you want to install wayfire-plugins-extra? [y/n]? y
+ case "$yn" in
+ yn=Y
+ break
+ '[' Y = Y ']'
+ check_download wayfire-plugins-extra
+ cd /home/amadeus/Projects/github.com/WayfireWM/wf-install
+ '[' '!' -d wayfire-plugins-extra ']'
+ '[' 0 = 1 ']'
+ cd wayfire-plugins-extra
+ git checkout origin/0.9.x
HEAD is now at 268e05d update all scripts to use pywayfire (#255)
+ cd /home/amadeus/Projects/github.com/WayfireWM/wf-install/wayfire-plugins-extra
+ PKG_CONFIG_PATH=:/opt/wayfire/lib/x86_64-linux-gnu/pkgconfig
+ meson setup build --prefix=/opt/wayfire -Dbuildtype=release -Db_lto=true
Directory already configured.

Just run your build command (e.g. ninja) and Meson will regenerate as necessary.
Run "meson setup --reconfigure to force Meson to regenerate.

If build failures persist, run "meson setup --wipe" to rebuild from scratch
using the same options as passed when configuring the build.
+ ninja -C build
ninja: Entering directory `build'
[1/28] Compiling C++ object src/libghost.so.p/ghost.cpp.o
FAILED: src/libghost.so.p/ghost.cpp.o 
c++ -Isrc/libghost.so.p -Isrc -I../src -I/opt/wayfire/include -I/usr/include/cairo -I/usr/include/libpng16 -I/usr/include/freetype2 -I/usr/include/pixman-1 -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/harfbuzz -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/libdrm -I/usr/include/libevdev-1.0/ -I/usr/include/libxml2 -flto=auto -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c++17 -O3 -DWLR_USE_UNSTABLE -DWAYFIRE_PLUGIN -Wno-unused-parameter -fPIC -pthread -MD -MQ src/libghost.so.p/ghost.cpp.o -MF src/libghost.so.p/ghost.cpp.o.d -o src/libghost.so.p/ghost.cpp.o -c ../src/ghost.cpp
In file included from /opt/wayfire/include/wayfire/plugins/ipc/ipc-activator.hpp:5,
                 from ../src/ghost.cpp:29:
/opt/wayfire/include/wayfire/plugins/ipc/ipc-helpers.hpp:10:10: fatal error: nlohmann/json.hpp: No such file or directory
   10 | #include <nlohmann/json.hpp>
      |          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
[8/28] Compiling C++ object src/libkeycolor.so.p/keycolor.cpp.o
ninja: build stopped: subcommand failed.
```

Brew:

```
❯ brew search nlohmann
==> Formulae
nlohmann-json ✔
```

--- a/meson.build
+++ b/meson.build
@@ -18,7 +18,6 @@ giomm = dependency('giomm-2.4', required: false)
 wayland_protos = dependency('wayland-protocols', version: '>=1.12')
 wayland_server = dependency('wayland-server')
 evdev = dependency('libevdev')
-json = dependency('yyjson', required: false)
 
 if get_option('enable_wayfire_shadows') == true
     wayfire_shadows = subproject('wayfire-shadows')

--- a/src/meson.build
+++ b/src/meson.build
@@ -58,14 +58,13 @@ magnifier = shared_module('mag', 'mag.cpp',
     dependencies: [wayfire],
     install: true, install_dir: join_paths(get_option('libdir'), 'wayfire'))
 
-if json.found()
 obs = shared_module('obs', 'obs.cpp',
-    dependencies: [wayfire, json],
+    dependencies: [wayfire],
     install: true, install_dir: join_paths(get_option('libdir'), 'wayfire'))
+
 pin_view = shared_module('pin-view', 'pin-view.cpp',
-    dependencies: [wayfire, json],
+    dependencies: [wayfire],
     install: true, install_dir: join_paths(get_option('libdir'), 'wayfire'))
-endif
 
 showrepaint = shared_module('showrepaint', 'showrepaint.cpp',
     dependencies: [wayfire],

